.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - docker-compose up -d
+
+install:
+  - docker-compose exec web sh -c "pip install tox-travis"
+
+script:
+  - docker-compose exec web sh -c "tox"
```

Comment:
Instead of starting the project as a daemon with `docker-compose up -d`, you should also be able to do this:

```bash
docker-compose run --rm web sh -c "tox"
```

With `run`, the databases will be started alongside automatically.